docs/source/examples/dispatch_tutorial.ipynb

```diff
@@ -0,0 +1,522 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "total_demand = 185  # [MW], sets the total demand [units of energy]\n",
+    "demand = (np.sin((hours*np.pi/n_hours*2+phase_shift))*-1+np.ones(N)*(base_shift+1))\n",
+    "\n",
+    "np.random.seed(1234) # sets the seed for repeatability\n",
+    "\n",
+    "noise = np.random.random(N)\n",
+    "demand += noise\n",
+    "\n",
+    "demand = demand/demand.sum() * total_demand  # rescale\n",
+    "\n",
+    "with plt.style.context(\"dark_background\"):\n",
+    "    plt.plot(hours, demand, color='cyan')\n",
+    "    plt.ylabel('Demand [MW]')\n",
+    "    plt.xlabel('Time [hr]')\n",
+    "    plt.grid(alpha=0.2)\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating the Dispatch Model\n",
+    "\n",
+    "\n",
+    "The simplest model in `osier` requires two things\n",
+    "\n",
+    "1. A **net** demand profile (i.e., if you want to model renewable energy, you should provide a production curve for the given time period of interest).\n",
+    "\n",
+    "2. A list of `osier.Technology` objects that can meet the specified demand.\n",
+    "\n",
+    "In order to solve the model, simply call `DispatchModel.solve(solver=\"SOLVER_NAME\")`. This function returns `None`, but you can check that it succeeded by printing `DispatchModel.objective`.\n",
```

Comment:
Since this tutorial represents a new user's first introduction to DispatchModel, explicitly state here or elsewhere in this tutorial that DispatchModel minimizes total cost. I think this is only implied below when you calculated variable cost.